src/utils.rs

```diff
@@ -173,3 +173,15 @@ where
         obj.add_prefix(prefix);
     }
 }
+
+macro_rules! skip_fail {
```

Comment:
I don't like this macro: it's too dependent of the context.

Reply:
should i change the name or just remove the macro?